### PR TITLE
update azure-ipi-compact workflow with simple way

### PR DESCRIPTION
**Pull request by jinyunma**
update azure-ipi-compact workflow in simple way, similar with https://github.com/openshift/release/pull/36016

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-azure

**Comment by jinyunma** [comment]
@jianlinliu PTAL, thanks.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-azure-upi
+    workflow: cucushift-installer-rehearse-azure-ipi-compact
```
Comment (MayXuQQ) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml:42]: whether this change need be merged?

**Pull request merged**